builtin/logical/mysql/path_config_connection.go

```diff
@@ -58,6 +71,12 @@ func (b *backend) pathConnectionWrite(
 	return nil, nil
 }
 
+type connectionConfig struct {
+	// Deprecate "value" in coming releases
+	ConnectionString   string `json:"connection_string" json:"value"`
```

Comment:
@jefferai It picks up the right most one. I see what you are suggesting. I'll do it.